docs/gitlab-enterprise-connector.md

```diff
@@ -62,6 +62,12 @@ docker run --pull=always --restart=always \
         leanixacrpublic.azurecr.io/vsm-gitlab-broker
 ```
 
```

Comment:
@vg-leanix could you please review the new section? Thanks.

Reply:
LGTM - thanks :)